tests/test_astronomer_commander.py

```diff
@@ -23,3 +23,35 @@ def test_astronomer_commander_deployment(kube_version):
     assert "quay.io/astronomer/ap-commander:0.25.2" in jmespath.search(
         "spec.template.spec.containers[*].image", doc
     )
+    assert (
+        doc["spec"]["template"]["spec"]["containers"][0]["env"][6]["name"]
```

Comment:
rather than using `[6]` here we should use `jmespath.search`

Reply:
Something like https://github.com/astronomer/astronomer/blob/533d047afc/tests/test_privateca.py#L57-L60